tls/s2n_fingerprint.c

```diff
@@ -235,7 +236,7 @@ static S2N_RESULT s2n_fingerprint_ja3(struct s2n_client_hello *ch,
         struct s2n_stuffer *output, uint32_t *output_size, struct s2n_hash_state *hash)
 {
     RESULT_ENSURE_REF(ch);
-    RESULT_ENSURE(!ch->sslv2, S2N_ERR_PROTOCOL_VERSION_UNSUPPORTED);
+    RESULT_ENSURE(ch->version > S2N_SSLv2, S2N_ERR_PROTOCOL_VERSION_UNSUPPORTED);
```

Comment:
I assume we are intentionally checking more here (i.e. ch->version isn't an unknown protocol version).